exporter_test.go

```diff
@@ -1307,6 +1323,13 @@ version = "4.5.6"
 	})
 }
 
+func checkEntrypoint(t *testing.T, image *fakes.Image, entrypointPath string) {
```

Comment:
I was thinking about our discussion earlier, and I think you're right, there's no need to put the `filepath.Join` in the argument. I think if the call were to read e.g., `assertHasEntrypoint("cnb", "lifecycle", "launcher")` that would be clear enough. It was really the omission of the leading `cnb` that was making it a bit hard to follow before.

Reply:
We changed the function name and kept the `filepath.Join` as is.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/520/commits/7b24df17bb72f57cfba08d780795cbaba5aaddca.